assets/sass/_ie.scss

```diff
@@ -12,6 +12,16 @@
     }
   }
 
+  .auth {
+    max-height: 500px;
+    overflow: auto;
+  }
+
+  .footer {
```

Comment:
the footer on the loginpage is 
![image](https://cloud.githubusercontent.com/assets/6729727/15023945/8dc71c76-11e7-11e6-844b-09487d878fa9.png)

if we don't add the footer class, the footer shows up in the middle of the widget which looks funny